contrib/!lang/ruby/packages.el

```diff
@@ -32,6 +32,14 @@
     :config (add-hook 'enh-ruby-mode-hook
                       (lambda () (rbenv-use-corresponding)))))
 
+(defun ruby/init-chruby ()
+  "Initialize chruby mode"
+  (use-package chruby
+    :defer t
+    :init (chruby)
```

Comment:
You don't need to call chruby here.
`defer t` enable deferred loading so `chruby` will be loading when the first ruby file is opened.